R/md.R

```diff
@@ -123,8 +126,44 @@ convert_code <- function(text) {
     text,
     rex::rex(
       "\\code{",
-      capture(one_or_more(none_of("{}%"))),
+      capture(one_or_more(none_of("{}"))),
```

Comment:
How does your code handle `.Rd` comments at EOL (prefixed by `%`)?

Reply:
I didn't test this, I *think* it will just keep the comment and when the code is split back into separate lines it wiil still be and the EOL, but I can check.